src/indexer/services/evmIndexer/ofac/index.ts

```diff
@@ -0,0 +1,45 @@
+import url from "url"
+
+type ChainAnalysisIdentIfication = {
+  category: string
+  name: string
+  description: string
+  url: string
+}
+
+type ChainAnalysisResponse = {
+  identifications: Array<ChainAnalysisIdentIfication> | []
+}
+
+enum AddressStatus {
+  OFAC = "ofac",
+}
+
+export class OfacComplianceService {
+  private chainAnalisysUrl: string | undefined
+  private chainAnalisysApiKey: string | undefined
+
+  constructor(chainAnalysisUrl?: string, chainAnalisysApiKey?: string) {
+    this.chainAnalisysUrl = chainAnalysisUrl
+    this.chainAnalisysApiKey = chainAnalisysApiKey
+  }
+
+  public async checkSanctionedAddress(address: string): Promise<string> {
+    if (!this.chainAnalisysUrl || !this.chainAnalisysApiKey) throw new Error("Chain Analysis credentials not found")
```

Comment:
This makes little sense. 
Check that when initializing the OfacService in the `init()` function in the `index.ts` in the indexer and don't start the service or throw an error depending on what you want.

Reply:
And make the credentials non-optional in the constructor

Reply:
Do that for CoinMarketCap service as well as I guess I missed it there.